### PR TITLE
Update changelog for 51 release

### PR DESCRIPTION
**Pull request by McdonaldSeanp**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,31 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+- Vanagon now assumes anything that is not listed as an archive is just
+  a file, and thus NON-ARCHIVE-EXTENSIONS no longer needs to be updated
+  for new extentions.
+- Vanagon now searches for full windows name in place of only searching
+  for "win"
+
+### Fixed
+- Builds will halt if any git command fails
+- Hardware engine will release locks on failures
+- Removed deprecated windwos functions: drive_root and
```
Comment (shermdog) [inline CHANGELOG.md]: *Windows

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,31 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
 
 ## [Unreleased]
-## [0.5.0] - 2015-12-22
+
+## [0.5.1] - 2016-2-04
+### Changed
+- Vanagon now assumes anything that is not listed as an archive is just
+  a file, and thus NON-ARCHIVE-EXTENSIONS no longer needs to be updated
+  for new extentions.
+- Vanagon now searches for full windows name in place of only searching
```
Comment (melissa) [inline CHANGELOG.md]: I wouldn't include this. This shouldn't impact anyone using `0.5.0`

Comment (McdonaldSeanp) [inline CHANGELOG.md]: Coo

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,7 +5,31 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+  for "win"
+
+### Fixed
+- Builds will halt if any git command fails
+- Hardware engine will release locks on failures
+- Removed deprecated windows functions: drive_root and
```
Comment (melissa) [inline CHANGELOG.md]: Same with this. I think this fixed functionality that wasn't available in the earlier release

Comment (McdonaldSeanp) [inline CHANGELOG.md]: also coo

**Comment by stahnma** [comment]
Once travis is done, Sean can merge this himself.

**Comment by melissa** [comment]
:+1:

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,7 +5,27 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
-## [0.5.0] - 2015-12-22
+
+## [0.5.1] - 2016-2-04
```
Comment (melissa) [inline CHANGELOG.md]: mmm, is the release date no longer correct? Do we care? (damn you travis!)

Comment (mckern) [inline CHANGELOG.md]: No it's still correct. This is going out tonight so we can unblock @heathseals.

**Comment by haus** [comment]
Building nugets seems pretty featurey. Any reason this wasn't 0.6.0?

**Comment by mckern** [comment]
I called that; didn't think it was a big enough feature to bump the Y; thought we'd save that for the rest of the Windows work landing.

**Comment by haus** [comment]
cool

**Pull request merged**